lms/djangoapps/certificates/api.py

```diff
@@ -949,3 +979,173 @@ def clear_pii_from_certificate_records_for_user(user):
         None
     """
     GeneratedCertificate.objects.filter(user=user).update(name="")
+
+
+def get_cert_history_for_course_id(course_id):
```

Comment:
Nit: Would be nice to have type hints or to specify the type in a docstring. A "course id" can have loaded meaning in the codebase. It could be a string representation of a course run id, but sometimes it ends up being an actual CourseLocator/CourseKey.